### PR TITLE
feat: creating and positioning a value dropdown component

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
+ apply non-inheritable

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue [hunk 1]
```diff
@@ -73,39 +75,52 @@ watch(() => props.inherit, () => {
...
-                <p-select-dropdown use-fixed-menu-style
-                                   :is-filterable="!props.inherit"
-                                   :multi-selectable="props.selectionType === 'MULTI'"
-                                   :handler="props.menuHandlers"
-                                   :selected="props.selected"
-                                   :invalid="!!state.errorMessage"
-                                   @update:selected="handleUpdateSelected"
-                                   @update:visible-menu="handleUpdateVisibleMenu"
+            <div>
+                <div v-if="props.label !== 'Cost Tag' || props.label !== 'Cost Additional Info'"
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:87]: suggestion: How about receiving 'widget filter key(e.g. filters.provider)' via props?
I'm concerned that `label` can be customized in the widget config😅

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue [hunk 2]
```diff
@@ -73,39 +75,52 @@ watch(() => props.inherit, () => {
...
+                    <p-icon-button v-if="props.deletable"
+                                   class="delete-button"
+                                   shape="square"
+                                   style-type="negative-secondary"
+                                   name="ic_delete"
+                                   @click="emit('delete')"
+                    />
+                </div>
+                <!-- TODO: will be updated-->
+                <dashboard-cost-widget-value-option-dropdown
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:117]: question: Is this module only usable in the **cost** widget?🧐

**Review by yuda110 — APPROVED** [review]
LGTM! I left few comments!

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue (hunk 1 above)
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:87]: nitpick: please add comment like 'HACK'

**Review by WANZARGEN — APPROVED** [review]
Thx for your effort! Looks good to me

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue (hunk 2 above)
Comment (skdud4659) [reply apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue:117]: This component is named for the cost-tag and cost-additional-info options!

**Pull request merged**